package/raphael.js

```diff
@@ -5711,6 +5717,11 @@
         next,
         prev,
         timestamp,
+        tempDiff,
+        tempDiff1,
+        tempDiff2,
+        tempDiff3,
+        change,
```

Comment:
The use of so many variables here simply makes this function hard to read. Try moving this to a function that takes Params as attributes and then returns true or false for change and if needed mutates the parameters. 

You can call this function `calculateAttrDiff`